.github/workflows/release.yml

```
@@ -1,10 +1,9 @@
 name: Release
 
 on:
+  workflow_dispatch:
   push:
     branches: [ $default-branch ]
```

Apparently $default-branch only gets resolved in workflow templates
```suggestion
    branches: [ master ]
```